Add 'kernel_type' attribute to programmatic datasets

We need to adjust the following programmatic sample datasets:

- `svm-model-generate.json`
- `svr-model-generate.json`

to incorporate the `sv_kernel_type` attribute.  Then, we need to adjust the [Data Attributes](https://github.com/jeff1evesque/machine-learning#data-attributes) section within the `README.md`.

**Note:** this issue compliments #1916.

In addition to the above stated issue, it is likely that since the sample datasets are not properly defined, we are getting the following unit test error traceback:

```bash
vagrant@vagrant-ubuntu-trusty-64:~$ cd /vagrant/test
vagrant@vagrant-ubuntu-trusty-64:/vagrant/test$ ls
config  programmatic_interface  Puppetfile  pytest.ini
vagrant@vagrant-ubuntu-trusty-64:/vagrant/test$ sudo pip install pytest
Requirement already satisfied (use --upgrade to upgrade): pytest in /usr/local/l
ib/python2.7/dist-packages
Cleaning up...
vagrant@vagrant-ubuntu-trusty-64:/vagrant/test$ ls
config  programmatic_interface  Puppetfile  pytest.ini
vagrant@vagrant-ubuntu-trusty-64:/vagrant/test$ py.test
============================= test session starts ==============================

platform linux2 -- Python 2.7.6, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
rootdir: /vagrant/test, inifile: pytest.ini
collected 4 items

programmatic_interface/pytest_session.py FFFF

=================================== FAILURES ===================================

________________________________ check_data_new ________________________________


    def check_data_new():
        """@check_data_new

        This method tests the 'data_new' session.

        """

>       assert requests.post(
            endpoint_url,
            headers=headers,
            data=get_sample_json('svm-data-new.json')
        )
E       assert <Response [500]>
E        +  where <Response [500]> = <function post at 0x7f4856db3de8>('http://l
ocalhost:5000/load-data/', headers={'Content-Type': 'application/json'}, data='{
"properties": {"svm_title": "sample_title", "svm_dataset_type": "json_string", "
svm_session": "data_new"}, "dataset"...": 427, "indep-variable-5": 75.45, "indep
-variable-2": 101.21, "indep-variable-3": 0.832, "indep-variable-1": 22.67}}}')
E        +    where <function post at 0x7f4856db3de8> = requests.post
E        +    and   '{"properties": {"svm_title": "sample_title", "svm_dataset_t
ype": "json_string", "svm_session": "data_new"}, "dataset"...": 427, "indep-vari
able-5": 75.45, "indep-variable-2": 101.21, "indep-variable-3": 0.832, "indep-va
riable-1": 22.67}}}' = get_sample_json('svm-data-new.json')

programmatic_interface/pytest_session.py:69: AssertionError
______________________________ check_data_append _______________________________


    def check_data_append():
        """@check_data_append

        This method tests the 'data_append' session.

        """

>       assert requests.post(
            endpoint_url,
            headers=headers,
            data=get_sample_json('svm-data-append.json')
        )
E       assert <Response [500]>
E        +  where <Response [500]> = <function post at 0x7f4856db3de8>('http://l
ocalhost:5000/load-data/', headers={'Content-Type': 'application/json'}, data='{
"properties": {"svm_session_id": "1", "svm_title": "sample_title", "svm_dataset_
type": "json_string", "svm_session":...": 235, "indep-variable-5": 64.45, "indep
-variable-2": 92.22, "indep-variable-3": 0.356, "indep-variable-1": 24.32}]}}')
E        +    where <function post at 0x7f4856db3de8> = requests.post
E        +    and   '{"properties": {"svm_session_id": "1", "svm_title": "sample
_title", "svm_dataset_type": "json_string", "svm_session":...": 235, "indep-vari
able-5": 64.45, "indep-variable-2": 92.22, "indep-variable-3": 0.356, "indep-var
iable-1": 24.32}]}}' = get_sample_json('svm-data-append.json')

programmatic_interface/pytest_session.py:83: AssertionError
_____________________________ check_model_generate _____________________________


    def check_model_generate():
        """@check_model_generate

        This method tests the 'model_generate' session.

        """

>       assert requests.post(
            endpoint_url,
            headers=headers,
            data=get_sample_json('svm-model-generate.json')
        )
E       assert <Response [500]>
E        +  where <Response [500]> = <function post at 0x7f4856db3de8>('http://l
ocalhost:5000/load-data/', headers={'Content-Type': 'application/json'}, data='{
"properties": {"svm_session_id": "1", "svm_model_type": "classification", "svm_s
ession": "model_generate"}}')
E        +    where <function post at 0x7f4856db3de8> = requests.post
E        +    and   '{"properties": {"svm_session_id": "1", "svm_model_type": "c
lassification", "svm_session": "model_generate"}}' = get_sample_json('svm-model-
generate.json')

programmatic_interface/pytest_session.py:97: AssertionError
_____________________________ check_model_predict ______________________________


    def check_model_predict():
        """@check_model_predict

        This method tests the 'model_predict' session.

        """

>       assert requests.post(
            endpoint_url,
            headers=headers,
            data=get_sample_json('svm-model-predict.json')
        )
E       assert <Response [500]>
E        +  where <Response [500]> = <function post at 0x7f4856db3de8>('http://l
ocalhost:5000/load-data/', headers={'Content-Type': 'application/json'}, data='{
"properties": {"svm_model_id": "1", "prediction_input[]": ["22.22", "96.24", "33
8", "72.55", "0.001", "28", "0.678"], "svm_session": "model_predict"}}')
E        +    where <function post at 0x7f4856db3de8> = requests.post
E        +    and   '{"properties": {"svm_model_id": "1", "prediction_input[]":
["22.22", "96.24", "338", "72.55", "0.001", "28", "0.678"], "svm_session": "mode
l_predict"}}' = get_sample_json('svm-model-predict.json')

programmatic_interface/pytest_session.py:111: AssertionError
=========================== 4 failed in 0.50 seconds ===========================

vagrant@vagrant-ubuntu-trusty-64:/vagrant/test$ exit
logout
Connection to 127.0.0.1 closed.
```

--- a/interface/static/data/json/programmatic_interface/svm/svm-data-append.json
+++ b/interface/static/data/json/programmatic_interface/svm/svm-data-append.json
@@ -1,9 +1,9 @@
 {
     "properties": {
-        "svm_title": "sample_title",
-        "svm_dataset_type": "json_string",
-        "svm_session": "data_append",
-        "svm_session_id": "1"
+        "title": "sample_title",
+        "dataset_type": "json_string",
+        "session": "data_append",
+        "session_id": "1"
     },
     "dataset": {
         "dep-variable-1": [

--- a/interface/static/data/json/programmatic_interface/svm/svm-data-new.json
+++ b/interface/static/data/json/programmatic_interface/svm/svm-data-new.json
@@ -1,8 +1,8 @@
 {
     "properties": {
-        "svm_title": "sample_title",
-        "svm_dataset_type": "json_string",
-        "svm_session": "data_new"
+        "title": "sample_title",
+        "dataset_type": "json_string",
+        "session": "data_new"
     },
     "dataset": {
         "dep-variable-1": [

--- a/interface/static/data/json/programmatic_interface/svm/svm-model-generate.json
+++ b/interface/static/data/json/programmatic_interface/svm/svm-model-generate.json
@@ -2,7 +2,7 @@
     "properties": {
         "session": "model_generate",
         "session_id": "1",
-        "model_type": "classification"
+        "model_type": "classification",
         "sv_kernel_type": "rbf"
     }
 }

--- a/interface/static/data/json/programmatic_interface/svm/svm-model-generate.json
+++ b/interface/static/data/json/programmatic_interface/svm/svm-model-generate.json
@@ -1,7 +1,8 @@
 {
     "properties": {
-        "svm_session": "model_generate",
-        "svm_session_id": "1",
-        "svm_model_type": "classification"
+        "session": "model_generate",
+        "session_id": "1",
+        "model_type": "classification"
+        "sv_kernel_type": "rbf"
     }
 }

--- a/interface/static/data/json/programmatic_interface/svm/svm-model-predict.json
+++ b/interface/static/data/json/programmatic_interface/svm/svm-model-predict.json
@@ -1,7 +1,7 @@
 {
     "properties": {
-        "svm_session": "model_predict",
-        "svm_model_id": "1",
+        "session": "model_predict",
+        "model_id": "1",
         "prediction_input[]": [
             "22.22",
             "96.24",

--- a/interface/static/data/json/programmatic_interface/svr/svr-model-generate.json
+++ b/interface/static/data/json/programmatic_interface/svr/svr-model-generate.json
@@ -3,5 +3,6 @@
         "session": "model_generate",
         "session_id": "1",
         "model_type": "regression"
+        "sv_kernel_type": "poly"
     }
 }

--- a/interface/static/data/json/programmatic_interface/svr/svr-model-generate.json
+++ b/interface/static/data/json/programmatic_interface/svr/svr-model-generate.json
@@ -2,7 +2,7 @@
     "properties": {
         "session": "model_generate",
         "session_id": "1",
-        "model_type": "regression"
+        "model_type": "regression",
         "sv_kernel_type": "poly"
     }
 }

